packages/account/src/Sections/Profile/PhoneVerification/cancel-phone-verification-modal.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Text } from '@deriv-com/quill-ui';
+import { Modal } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { useHistory } from 'react-router';
+import React from 'react';
+
+type TCancelPhoneVerificationModal = {
+    show_cancel_verification_modal: boolean;
```

Comment:
would it be better to crate more evident got boolean value name, something like should_show_.....
you can ignore this comment :)